app-datepicker.html

```diff
@@ -1002,9 +1002,10 @@
             && window.Intl.DateTimeFormat().resolvedOptions().locale)
             || 'en-US');
           var _amy = new window.Intl.DateTimeFormat(_locale, {
+            timeZone: 'UTC',
```

Comment:
Of course: 

If you dont set timeZone, Intl will use client timezone: for example, in my case Europe/Madrid, or America/Buenos_Aires or Africa/Dakar, or Europe/Bucharest....

When creating a new Date.UTC, it is truncated, there is no offset (GMT), so when convert for example 2000-01-01 (2000-01-01T00:00:00Z) this will be transformed to local timezone (adding TZ offset with sign). In my case there is no problem because I'm 1 or 2 hours to the East, so all my dates are 1 or 2 hours ahead but any TZ with negative offset will be transformed substracting TZ offset, for example for America/Buenos_Aires TZ 2000-01-01 in UTC is the same as 1999-12-31 but the date is the date, no matter TZ of the client.